scripts/build/build.sh

```diff
@@ -39,7 +39,13 @@ $REPOROOT/scripts/compile/compile.sh
 
 # Run tests on the stage2 output
 header "Testing stage2..."
+START_TOOLS=$DOTNET_TOOLS
```

Comment:
Ok. This pattern was already used in a number of the other refactored scripts so I mirrored it here. I don't think it's needed because each script is launched in a new shell process anyway.